src/sass/components/_about.scss

```diff
@@ -0,0 +1,151 @@
+.section--about {
+  padding-bottom: 40px;
+
+	.section__sidebar {
+    display: flex;
```

Comment:
Pls, apply `prettier`

Reply:
and `prettier:fix"` script in `package.json` as well

Reply:
done in https://github.com/kottans/kottans.github.io/pull/164